src/ess/reflectometry/tools.py

```diff
@@ -170,28 +170,43 @@ def _interpolate_on_qgrid(curves, grid):
 
 def scale_reflectivity_curves_to_overlap(
     curves: Sequence[sc.DataArray],
-    return_scaling_factors=False,
-) -> list[sc.DataArray] | list[sc.scalar]:
+    critical_edge_interval: tuple[sc.Variable, sc.Variable] | None = None,
+) -> tuple[list[sc.DataArray], list[sc.scalar]]:
     '''Make the curves overlap by scaling all except the first by a factor.
     The scaling factors are determined by a maximum likelihood estimate
     (assuming the errors are normal distributed).
 
+    If :code:`critical_edge_interval` is provided then all curves are scaled.
+
     All curves must be have the same unit for data and the Q-coordinate.
 
     Parameters
     ---------
     curves:
         the reflectivity curves that should be scaled together
-    return_scaling_factor:
-        If True the return value of the function
-        is a list of the scaling factors that should be applied.
-        If False (default) the function returns the scaled curves.
+    critical_edge_interval: optional,
```

Comment:
```suggestion
    critical_edge_interval:
```
I don't think there should be anything after the colon. The type-hint plugin will fill in the type here.